README.txt

```diff
@@ -4,14 +4,54 @@ Barrett Technology
 
 
 Libbarrett is a real-time controls library written in C++ that runs Barrett
-Technology's products, including the WAM Arm and the BH8-280 BarrettHand. For
+Technology's products, including the WAM Arm, Proficio and the BH8-280 BarrettHand. For
 support, please email:
     support@barrett.com
 
-To build and install libbarrett, run:
-    cmake .
-    make
-    sudo make install
+To build and install libbarrett
+
+Source based install
+
+Pre-requisites:
+
+$ sudo apt-get install python-dev python-argparse
+$ sudo apt-get install libeigen2-dev libboost-all-dev libgsl0-dev
+$ sudo apt-get install libxenomai-dev libxenomai1
+$ wget http://web.barrett.com/svn/libbarrett/dependencies/libconfig-1.4.5-PATCHED.tar.gz
+$ tar -xf libconfig-1.4.5-PATCHED.tar.gz
+$ cd libconfig-1.4.5
+$ ./configure && make && sudo make install
+$ cd ../
+$ rm -rf libconfig-1.4.5 libconfig-1.4.5-PATCHED.tar.gz
+
+Download and install libbarrett:
+
+$ cd ~/
+$ git clone https://github.com/BarrettTechnology/libbarrett.git
```

Comment:
I think we need a "git checkout" command here to specify a particular branch, else it may refer to origin/master by default.